src/game/scenes/arena.c

```diff
@@ -1118,6 +1118,10 @@ void arena_input_tick(scene *scene) {
         controller_free_chain(p2);
     }
 
+    if(is_netplay(scene->gs)) {
+        return;
```

Comment:
Maybe put a comment on what this does :/

Reply:
Added.